.codesandbox/ci.json

```diff
@@ -1,5 +1,5 @@
 {
   "packages": ["packages/*", "design-system/packages/*"],
   "sandboxes": ["/tests/sandbox"],
-  "node": "18"
+  "node": "20"
```

Comment:
Code sandbox does not yet support node 20 https://codesandbox.io/docs/learn/sandboxes/ci should we keep this at 18 for now?

Reply:
```suggestion
  "node": "18"
```